modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/operation/DefaultExecutionMediator.java

```diff
@@ -139,7 +130,6 @@ public DefaultExecutionMediator(ExtensionModel extensionModel,
       Mono<Object> result;
       if (beforeExecutionResult.isOk()) {
         result = from(withContextClassLoader(getClassLoader(context.getExtensionModel()), () -> executor.execute(context)))
-            .map(value -> transform(context, value))
```

Comment:
why this?

Reply:
the concept of "transformers" in the mediator wasn't fitting correctly and paging provider was their unique customer,so is being deleted.